MusicTaste/Pods/AFNetworking/AFNetworking/AFHTTPSessionManager.m

```diff
@@ -0,0 +1,361 @@
+// AFHTTPSessionManager.m
```

Comment:
As a best practice we generally add the Pods subdirectory to gitignore since it is external code that does not need reviewing

Reply:
This one is [debatable though](https://stackoverflow.com/questions/9446644/what-goes-into-your-gitignore-if-youre-using-cocoapods) so let's keep it in the repo